code/ecdsap256/Hacl.Impl.P256.LowLevel.RawCmp.fst

```diff
@@ -29,3 +30,14 @@ inline_for_extraction noextract
 val eq_0_u64: a: uint64 -> r:bool{r == (uint_v a = 0)}
 
 let eq_0_u64 a = eq_u64_nCT a (u64 0)
+
+
+
+inline_for_extraction noextract
+val unsafe_bool_of_u64 (x: uint64 { v x == 0 \/ v x == pow2 64 - 1 }):
+  (b:bool { b <==> v x == 0 })
+
+let unsafe_bool_of_u64 x = 
```

Comment:
you can write it as `FStar.UInt64.(u64_to_UInt64 x =^ 0uL)`, in this way the extracted code will look nicer.